components/RuiBreadcrumbs/src/RuiBreadcrumbs.css.ts

```diff
@@ -2,127 +2,258 @@ import {css} from 'lit-element';
 
 
 /**
-	*  RuiBreadcrumbs Variables
- 	*  Added colour to prevent empty block
-*/
-
+ *  RuiBreadcrumbs Variables
+ */
 export const variables = css`
 	:host {
-		/* BASE STYLES */
+		/**
+		 * @variable The padding on each breadcrumb link item
+		 */
+		--rui-breadcrumbs__padding: 0 4px;
+
+		/**
+		 * @variable The padding on each breadcrumb link item when focused
+		 */
+		--rui-breadcrumbs__padding-focus: 0 3px;
```

Comment:
@jnelssonsmith can we not resolve this with `box-sizing: border-box` instead of having to specify separate padding?

Reply:
@ricominten unfortunately not as we cannot specify a fixed width/height, see [this SO answer](https://stackoverflow.com/questions/52242410/box-sizing-border-box-with-no-declared-height-width) for more details. 

Solutions are either changing padding as we do here, or setting a transparent border at all whenever we don't want one to preserve the size. I'd say we keep the padding solution for now, and if we decide we prefer the border solution we'll raise an issue and implement later